lib/graphql_devise/mutations/register.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module GraphqlDevise
+  module Mutations
+    class Register < Base
+      argument :email,                 String, required: true
+      argument :password,              String, required: true
+      argument :password_confirmation, String, required: true
+      argument :confirm_url,           String, required: false
+
+      field :credentials,
+            GraphqlDevise::Types::CredentialType,
+            null:        true,
+            description: 'Authentication credentials. Null if after signUp resource is not active for authentication (e.g. Email confirmation required).'
+
+      def resolve(confirm_url: nil, **attrs)
+        resource = build_resource(attrs.merge(provider: provider))
+        raise_user_error(I18n.t('graphql_devise.resource_build_failed')) if resource.blank?
+
+        redirect_url = confirm_url || DeviseTokenAuth.default_confirm_success_url
+        if confirmable_enabled? && redirect_url.blank?
+          raise_user_error(I18n.t('graphql_devise.registrations.missing_confirm_redirect_url'))
+        end
+
+        check_redirect_url_whitelist!(redirect_url)
+
+        resource.skip_confirmation_notification! if resource.respond_to?(:skip_confirmation_notification!)
+
+        if resource.save
+          yield resource if block_given?
+
+          unless resource.confirmed?
```

Comment:
Will this consider `skip_confirmation_notification` ?

Reply:
Not really related, `skip_confirmation_notification` simply prevents regular devise email from going out. DTA does the same